ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -98,12 +86,35 @@ class TooltipView: UIView {
         messageLabel.text = message
         messageLabel.textAlignment = textAlignment
         messageLabel.isAccessibilityElement = false
-        addSubview(messageLabel)
+
+        if let titleLabel = titleLabel, let title = title {
+            titleLabel.text = title
+            titleLabel.textAlignment = textAlignment
+            titleLabel.isAccessibilityElement = false
+        }
+
+        addSubview(textContainer)
 
         // Shadow
```

Comment:
can adding shadow be its own private function. i think this function is getting way to big